### ISSUE TITLE
Creating new user: "Invalid username or password"

### ISSUE THREAD

**Issue by OfficialCRUGG** [box/mojito#472]
Hey there!
I just installed Mojito and deleted the admin user and wanted to create a new user. I used the following command:
```
java -jar mojito-cli-0.99.jar user-create  --username CRUGG --surname CRUGG --given-name CRUGG --common-name CRUGG --role ADMIN --password
```
and then entered my password (only letters and numbers). When I do that, it says "Invalid username or password". Can somebody help?

**Comment by aurambaj** [comment]
You need to be authenticated to create users and the CLI uses by default `admin` user to do so. If you removed it you won't be able to run commands without specifying another user.

If you have another user for which you know the password you can use it to re-create the `admin` user. 

Here are way of configuring the CLI to use a different user: https://www.mojito.global//docs/refs/configurations/#cli-configuration

In short you can do something like: 
```
java -Dl10n.resttemplate.authentication.username=myotheruser -D l10n.resttemplate.authentication.password=mypwd mojito-cli-0.99.jar user-create  --username CRUGG --surname CRUGG --given-name CRUGG --common-name CRUGG --role ADMIN --password
```

**Comment by aurambaj** [comment]
Another way if you have access to the DB is to find the deleted user, the name will be something like `deleted__{timestamp}__admin`. You can rename it to `admin` and set `enabled` flag to true.

Closing for now. Let me know if you need more help.
